shell/app/config-page/components/table-group/table-group.tsx

```diff
@@ -37,14 +37,7 @@ const TableBoard = (props: CP_TABLE_GROUP.ITableBoardProps) => {
           <div className="mb12 ml4">
             <Text props={description} type="Text" {...extraProps} />
           </div>
-          <Table
-            tableLayout="fixed"
-            props={table?.props}
-            data={table?.data}
-            operations={table?.operations}
-            {...extraProps}
-            type="Table"
-          />
+          <Table props={table?.props} data={table?.data} operations={table?.operations} {...extraProps} type="Table" />
```

Comment:
does antd table has this attribute? `type="Table"`

Reply:
it's not a antd Table